qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -0,0 +1,18 @@
+- numa_prealloc_threads:
+    no RHEL.6 RHEL.7 RHEL.8 RHEL.9.0 RHEL.9.1
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
```

Comment:
If i understand correctly,   it need `required_qemu = [7.2, )`

Reply:
Does this feature released with the guest-os or it depends on the hypervisor only ?  Be careful with the host version and guest version limitation. thanks!

Reply:
@yanan-fu Yes, my fault... this feature depends on QEMU